Cargo.toml

```diff
@@ -117,6 +117,9 @@ unicode-width = "0.1"
 # Used for cancellable wait on worker processes
 wait-timeout = "0.2"
 
+# Used to parse versions
+regex = "1.10"
+
```

Comment:
Nit: please consider keeping the list alphabetically sorted.

Reply:
Updated. :+1: